src/librustdoc/doctest.rs

```diff
@@ -964,6 +964,7 @@ impl Tester for Collector {
                 test_type: test::TestType::DocTest,
             },
             testfn: test::DynTestFn(Box::new(move || {
+                let _ = &config;
```

Comment:
This change was the only compatibility fix suggested by `-D edition-2021-compatibility` (I didn't use `cargo fix --edition` because I'm not sure how to use it with bootstrap). It seems unlikely to be necessary, but preserves the old drop order. Should I remove it?

Reply:
Nah, drop order doesn't matter here. There's no mutexes or anything.

Reply:
Ok, I dropped this change.